sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/AssignRoleToUserTest.java

```diff
@@ -65,10 +65,11 @@ public void testAssignRoleToUserFailure(
       String testName, boolean isAuthRequired, HttpStatus httpStatusCode) {
     getAuthToken(this, isAuthRequired);
     createUser();
-    createOrg();
     variable("userId", testContext.getVariable("userId"));
-    variable("organisationId", testContext.getVariable("organisationId"));
-
+    if(isAuthRequired) {
+      createOrg();
+      variable("organisationId", testContext.getVariable("organisationId"));
```

Comment:
Following statement can be part of createOrg().

      variable("organisationId", testContext.getVariable("organisationId"));